apify-docs-theme/src/theme/NavbarItem/NavbarNavLink.jsx

```diff
@@ -16,6 +17,9 @@ export default function NavbarNavLink({
     prependBaseUrlToHref,
     ...props
 }) {
+    const { navbar: { items = [] } } = useThemeConfig();
+    const { options: { subNavbar = [] } } = usePluginData('@apify/docs-theme');
+    const allItems = [...items, ...subNavbar];
```

Comment:
`subNavbar` is an object with properties "title" and "items" - the spread will probably not work as intended

Reply:
good catch